lib/describe-metadata-result.json

```diff
@@ -685,7 +685,13 @@
 		"metaFile": false,
 		"suffix": "lightningExperienceThemes",
```

Comment:
and by the way: I think this should then be: `"suffix": "lightningExperienceTheme"` (singular)